stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -726,71 +758,6 @@ public final class _DataStorage {
         }
     }
     
-    public static func ==(_ lhs: _DataStorage, _ rhs: _DataStorage) -> Bool {
```

Comment:
I didn't see where this code went; just the diff being confusing, or was this actually deleted?